src/generics.jl

```diff
@@ -83,7 +83,7 @@ the quadratic form is applied column-wise.
 """
 function quad(a::AbstractPDMat{T}, x::StridedMatrix{S}) where {T<:Real, S<:Real}
     @check_argdims dim(a) == size(x, 1)
-    quad!(Array{promote_type(T, S)}(uninitialized, size(x,2)), a, x)
+    quad!(Array{promote_type(T, S)}(Compat.undef, size(x,2)), a, x)
```

Comment:
This can actually just be `undef`, since `undef` is exported from Compat and we have a `using Compat`. The version of Compat in REQUIRE should be set to 0.61.0 for this as well.